addons/digest/data/digest_tips_data.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
     <data noupdate="1">
         <record id="digest_digest_default" model="digest.digest">
-            <field name="name">Your Odoo Periodic Digest</field>
+            <field name="name">Your Periodic Digest</field>
```

Comment:
Are we actually using the digest addon?

Reply:
yep, when running UDES gentoo its appears in the settings menu.